geometry_msgs/PoseStamped start_position
geometry_msgs/PoseStamped end_position
bool update_map
bool update_samples
---
nav_msgs/Path path
